### ISSUE TITLE
Is it possible to catch PanicException

### ISSUE THREAD

**Issue by ruiqic** [kevinheavey/solders#108]
Seems like the rust part of the code does not correctly handle some responses. This is using quicknode RPC btw. `pyo3_runtime` is not a valid module, so the `PanicException` can't be imported. Even so, using `except` that attempts to catch all still can't catch the "exception".  Any solutions? 

```console
Traceback (most recent call last):
  ...
  File "__/script.py", line 500, in __
    first_resp = await ws.recv()
  File "__/lib/python3.10/site-packages/solana/rpc/websocket_api.py", line 115, in recv
    return self._process_rpc_response(cast(str, data))
  File "__/lib/python3.10/site-packages/solana/rpc/websocket_api.py", line 398, in _process_rpc_response
    parsed = parse_websocket_message(raw)
pyo3_runtime.PanicException: unsupported type Some(-32608)
```

**Comment by kevinheavey** [comment]
What input caused the panic? We should avoid panicking in the library but maybe something slipped through

**Comment by kevinheavey** [comment]
Fixed by #113
